larray_editor/editor.py

```diff
@@ -933,8 +936,7 @@ def _open_file(self, filepath):
         try:
             session.load(filepath, names)
             self._reset()
-            self._add_arrays(session)
-            self._listwidget.setCurrentRow(0)
+            self._push_data(session)
```

Comment:
AFAICT, _add_arrays is not longer called. If this is indeed the case, it should be removed.